tests/data/common/app/grpc/app.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: grpc
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: greeter
+  template:
+    metadata:
+      labels:
+        app: greeter
+    spec:
+      containers:
+      - name: greeter
+        image: nginxkic/test-grpc-server:0.1
```

Comment:
should we publish the sources similarly to how we published the sources for the TCP and UDP servers? so we don't loose it

Reply:
I'm not too sure about publishing sources for this one as the code isn't written by our team members unlike TCP/UDP server one which @soneillf5 wrote.

Reply:
in the case of the backend, it's just a Dockerfile, right? we don't really modify the backend code?

Reply:
where does the test-grpc-server come from then ?